CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.8.1]
+### Uncategorized
```

Comment:
The "Uncategorized" category I introduced was only meant as a temporary header, as a request for manual intervention basically. We shouldn't actually use this category for anything if we can help it. In future versions of `@metamask/auto-changelog` it will be prevented by validation on release branches.
```suggestion
### Changed
```

Reply:
@Gudahtt that makes sense, I only left it this way to match the other case in the changelog where we had no direct entries, `8.1.7`, I can update